src/App.tsx

```diff
@@ -1,11 +1,25 @@
 import React from 'react';
+import { useSearchParams } from 'react-router-dom';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
+function getItems(itemsArr: string[], page: number, perPage: number) {
+  const indStart = (page - 1) * perPage;
+
+  return items.slice(indStart, indStart + perPage);
+}
```

Comment:
Use your first argument 
```suggestion
function getItems(itemsArr: string[], page: number, perPage: number) {
  const indStart = (page - 1) * perPage;

  return itemsArr.slice(indStart, indStart + perPage);
}
```